shielddb-core/src/test/java/nf/fr/k49/shielddb/core/ShieldDbBuilderForStringTest.java

```diff
@@ -0,0 +1,4 @@
+package nf.fr.k49.shielddb.core;
+
+public class ShieldDbBuilderForStringTest implements BaseShieldDBBuilderTest<String>{
+}
```

Comment:
Maybe can you add another test class with a custom object, in real use case we will mainly store custom object (maybe a simple User(firstname,lastname,birthdate))

Reply:
No problem. What about the Testclasses for String and Integer? Delete?

Reply:
I think you can keep it, they can represent some use cases too :slightly_smiling_face: